content/docs/ops/runbook/troubleshooting-logsearch.md

```diff
@@ -48,15 +48,19 @@ cp /var/vcap/jobs/ingestor_syslog/config/logstash.conf /tmp/logstash-restore.con
 
 Edit the `/tmp/logstash-restore.conf` and make the following changes:
 
-#### Remove the `redis` input and replace it with an `s3` input
+#### Update the "input" block to only contain s3
 ```
+input {
   s3 {
     bucket => ":bucket:"
     region => ":region:"
+    access_key_id => ":access_key_id:"
```

Comment:
Actually, torn. I clearly see value in having an instance profile. A restore is likely going to happen during the course of some event. Depending on stress levels, an operator could possibly leak credentials during this process if forced to inject them here, instead of just relying on the fact that the permissions needed to carry out the operation are already in place. Might be too paranoid a situation though.